tests/authorizations/test_emergency_contact_authorization.py

```diff
@@ -19,23 +16,52 @@ def test_emergency_contacts_POST(self, auth_headers):
             ],
         }
 
+        self.user_id = 1
+
+    def test_auth_header_is_required(self):
+        response = self.client.post(self.endpoint, json=self.newContact)
+
+        assert is_valid(response, 401)
```

Comment:
Let's change all of these over to
```python
assert response.status_code == 401
```

Reply:
I'm fine for leaving is_valid in for now. Flask-restful guarantees that it will be valid json response, If we switch off of flask-restful then... I'm fine either way here.